tests/loaders/test_loaders.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from unittest.mock import patch, Mock
+
+from gqlalchemy.loaders import (
+    DataLoader,
+    FileSystemHandler,
+    FileSystemTypeEnum,
+    NameMapper,
+    PyarrowDataLoader,
+    LocalFileSystemImporter,
+    get_data_loader,
+)
```

Comment:
Do we separate now `loaders.py` into multiple files under same directory? @BorisTasevski  @brunos252

Reply:
I would like to have it in subsequent PR @Josipmrden